internal/extproc/main/main.go

```diff
@@ -0,0 +1,78 @@
+package maincmd
```

Comment:
this needs to be moved to public package - i will take care of that once it's given a go